h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.ts

```diff
@@ -253,14 +254,17 @@ export class SummaryComponent implements OnInit {
     }
 
     // remove password
-
     if (results['operations']['opsPassword']) {
-      results['operations']['opsPassword'] = '*';
+      delete results['operations']['opsPassword'];
     }
 
     return results;
   }
 
+  escapeSpecialCharsForCLI(text) {
+    return text.replace(/[\]{}()*+?,\\^$|#\s]/g, '\\$&');
```

Comment:
Does `\$*` translate to some special var? And how did you come up with those special characters?

Reply:
$& is a "replacement" (placeholder for something to be substituted in) for the full match (all the matched text)